### ISSUE TITLE
Test for WCAG success criterion 2.4.1: Bypass blocks

### ISSUE THREAD

**Issue by hannolans** [quailjs/quail#175]
http://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-skip.html
A mechanism is available to bypass blocks of content that are repeated on the same Web page -- skip links and headings. 
## scope selector

Page (and a selector for regions, and a selector for blocks)
## Applicable

Always
## Failures

None defined
## Techniques
- [x] G1: Adding a link at the top of each page that goes directly to the main content area (#176 - implemented) 
- [ ] G123: Adding a link at the beginning of a block of repeated content to go to the end of the block
- [ ] G124: Adding links at the top of the page to each area of the content 
- [x] H69: Providing heading elements at the beginning of each section of content ( #198 )
- [x] H70: Using frame elements to group blocks of repeated material (#90)  AND H64: Using the title attribute of the frame and iframe elements (#65)
- [ ] SCR28: Using an expandable and collapsible menu to bypass block of content
## Relationship between techniques
- There are two main directions for this success criterion: 'links to skip blocks' or 'grouping blocks'.
- G124 seems a more advanced version of G1, so if G124 passes, G124 passes as well
- H70 and G123: 'repeated material' is difficult for testing, as we need more than one page for that
- H69, H70 AND SCR28 are about "Grouping blocks of repeated material in a way that can be skipped". Scope is 'group of blocks' (in Drupal called regions)
- G1, G123 and G124 are about 'Creating links to skip blocks'. Scope are blocks.
- H70 is only a pass for this success criterion if H64 also passes.
- H70 is a depreciated technology
- Because we have so many possible techniques and no failures, this test will most of the time pass. It's important that some of the most used techniques can give a pass. That would be G1 (skip link), and secondly, region headings
